new/server/models/user.js

```diff
@@ -3,13 +3,37 @@ const mongoose=require('mongoose');
 const Schema=mongoose.Schema;
 
 const userSchema=new Schema({
+    fname:{
+        type:String,
+        required:true
+    },
+    lname:{
+        type:String,
+        required:true
+    },
+    username:{
+        type:String,
+        required:true
+    },
     email:{
         type:String,
         required:true
     },
     password:{
         type:String,
         required:true
+    },
+    birthDate:{
+        type:String,
+        required:true
+    },
+    image:{
+        type:String,
+        required:true
+    },
+    role:{
+        type:String,
+        default:"admin"
```

Comment:
Default "user" not admin.

Also add description field, like I asked previously. Keep it `null` for now. A user will be able to add his description only after he is registered.